src/Microsoft.AspNetCore.Http.Features/IHttpBodyControlFeature.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Http.Features
+{
+    /// <summary>
+    /// Controls the IO behavior for the <see cref="IHttpRequestFeature.Body"/> and <see cref="IHttpResponseFeature.Body"/> 
+    /// </summary>
+    public interface IHttpBodyControlFeature
```

Comment:
Not sure about the naming BodyControl. Maybe IHttpBodyOptionsFeature, IHttpBodyIOFeature, IHttpBodyCapabilitiesFeature?